testing/python/metafunc.py

```diff
@@ -448,7 +448,10 @@ def test_idmaker_enum(self) -> None:
         enum = pytest.importorskip("enum")
         e = enum.Enum("Foo", "one, two")
         result = idmaker(("a", "b"), [pytest.param(e.one, e.two)])
-        assert result == ["Foo.one-Foo.two"]
+        assert result in [
```

Comment:
I described my preferred solution [here](https://github.com/pytest-dev/pytest/issues/8546#issuecomment-818662849), though shouldn't hold up this PR.